### PR TITLE
[ASan] Remove COMPILER_RT_ASAN_SHADOW_SCALE_DEFINITION.

### PR DESCRIPTION
**Pull request by Enna1**
Since the set of COMPILER_RT_ASAN_SHADOW_SCALE_DEFINITION is removed in commit 8421fa5d536aadf42c0e54c566bc439a40ebdb8e, 
cleanup the use of COMPILER_RT_ASAN_SHADOW_SCALE_DEFINITION.

### CODE REVIEW

**Review by thurstond — APPROVED** [review]

File: compiler-rt/lib/asan/CMakeLists.txt [hunk 1]
```diff
@@ -88,7 +88,7 @@ set(ASAN_CFLAGS ${SANITIZER_COMMON_CFLAGS})
 
 append_list_if(MSVC /Zl ASAN_CFLAGS)
 
-set(ASAN_COMMON_DEFINITIONS ${COMPILER_RT_ASAN_SHADOW_SCALE_DEFINITION})
+set(ASAN_COMMON_DEFINITIONS)
```
Comment (MaskRay) [inline compiler-rt/lib/asan/CMakeLists.txt:91]: Does `unset` work?

If this is removed, will the build break?

**Review by MaskRay — APPROVED** [review]

File: compiler-rt/lib/asan/CMakeLists.txt (hunk 1 above)
Comment (Enna1) [reply compiler-rt/lib/asan/CMakeLists.txt:91]: Thanks for pointing this out!
I tested both locally and in buildbot https://lab.llvm.org/buildbot/#/builders/37/builds/34788
Do you think `set(ASAN_COMMON_DEFINITIONS "")` is better?

Comment (MaskRay) [reply compiler-rt/lib/asan/CMakeLists.txt:91]: Yes, `""` might be better

Comment (Enna1) [reply compiler-rt/lib/asan/CMakeLists.txt]: done

**Pull request merged**